common-arcana.lic

```diff
@@ -166,6 +166,26 @@ module DRCA
     match
   end
 
+  # Returns true if preparing a spell, false otherwise.
+  # For the name of the spell, see `get_preparing_spell`
```

Comment:
Should this reference `spell_preparing`?

Reply:
Sure, I'll need to tweak both methods a bit to avoid the circular reference when I do that

Reply:
I only meant the comment, but if you spotted some other refactoring you want to do, sure thing

Reply:
Yep, saw an opportunity to reduce number of times I referenced XMLData.prepared_spell, thanks